Task-56509 :added the mainApplicationName property when creating an activity-stream

Amine0008 opened the pull request:
added the mainApplicationName property when creating an activity-stream so we can track the use of exo's main applications in mobile statistics

halaya committed:
TASK-35704: Create FB maintenance and update projects versions/dependencies

hakermi committed:
TASK-56601: Duplication and bad detection and display of tags (#1434)

Prior to this changes, when creation a tag and then press the space button each time will create another tag with number of spaces converted to &nbsp; as the matcher of tags pattern will consider it as a normal character
follows the tag text wich will duplicate the tag depends on the number of spaces that follows it and display it with the &nbsp; in the unified search when selecting tags, this because of the returned html content of the
activity when searching for tags, This PR will sanitize the content to correctly unescape the &nbsp; and replacing it by spaces to allow the tags pattern matcher to work as expected.

Amine0008 committed:
added the mainApplicationName property when creating activity-stream

ghost commented:
### CodeSee Review Map:
[![Review these changes using an interactive CodeSee Map](https://s3.us-east-2.amazonaws.com/maps.codesee.io/images/github/Meeds-io/social/1435/3d270dbd/e0d1dbd70972710bae9680b92eb7eefcd05c4d0a.svg)](https://app.codesee.io/r/reviews?pr=1435&src=https%3A%2F%2Fgithub.com%2FMeeds-io%2Fsocial)

Review [in an interactive map](https://app.codesee.io/r/reviews?pr=1435&src=https%3A%2F%2Fgithub.com%2FMeeds-io%2Fsocial)

View [more CodeSee Maps](https://app.codesee.io/r/maps?src=https%3A%2F%2Fgithub.com%2FMeeds-io%2Fsocial)

#### Legend
![CodeSee Map Legend](https://app.codesee.io/Legend_v2.png)

boubaker commented on webapp/portlet/src/main/webapp/vue-apps/activity-stream/main.js:
```diff
@@ -45,6 +45,9 @@ const urls = [
 ];
 
+//used only in mobile statistics
+eXo.env.portal.mainApplicationName = 'Social';
```
```suggestion
eXo.env.portal.mainApplicationName = 'activityStream';
```

It's better to add a name that you can add in i18N File and that is significative. To know if you want to use `social` (one single module name for all social apps) or `activityStream` + `peopleList` + `spaceList` + (moduleName per Application) ... could you please ask @Samuel-Renault-eXo or Morgan

BTW, could you please define `analytics.field.label.APP_NAME` in each project inside the file `locale.portlet.Analytics_en.properties` to allow internationalize the labels. (Each addon define this same file, so you will have to add a specific label in each file for each addon)

hakermi committed:
TASK-56601: Tags are duplicated when they are ending with white-spaces (#1439)

Prior to this change, when creating a tag and then add a whitespace right after it it will be recognized as a new tag and then duplicated, this due to the fact that the &nbsp; char are converted to and space
and recognized in java pattern as a char NBSP and so will be considered as a part fo the tagtext.
This PR should remove this weird char from the tag text to avoid duplicating it

ahamdi commented:
@Amine0008 Could you check the review ?

hakermi committed:
TASK-56832: Unescaped html characters in favorite activity titles (#1442)

Prior to this change the html content of the activity title are displayed  without escaping the special html chars,
This PR uses the custom html-sanitized directive to correctly unescape these chars

azayati committed:
Task-55805: [Accessibility - Contrast - 3.2] Change the default primary color (#1441)

sofyenne committed:
Task-56602: Fix icons misalignment in Toolbar. (#1436)

* Task-56602: Copy of [AUTH][UXFT] TopBar_BUG01.2: Favorite icon alignment

problem: bad alignement of icon favorite

* Update webapp/portlet/src/main/webapp/vue-apps/favorites-list-top-bar/components/ExoTopBarFavorites.vue

Co-authored-by: Boubaker Khanfir <boubaker.khanfir@exoplatform.com>

Co-authored-by: Boubaker Khanfir <boubaker.khanfir@exoplatform.com>

hakermi committed:
TASK-56851: Add and ellipsis to space name in space Access portlet when the space name is very long (#1443)

Prior to this change, On mobile screen, when the space where the user has been invited has a long name then it's not well displayed in the space access page and the name overflows the spaceblock access area.
THis Pr should add the class text-truncate after adding it to the dedicated portlet style in platform-ui to truncate the name and display an ellipsis with a html tooltip

Amine0008 committed:
changed the mainApplicationName property from Social to social to utilise the already existing tranlations of the analytics.social property

ahamdi committed:
Merge branch 'feature/maintenance' into task-56509

azayati reviewed (changes requested):
Please update the title in order to use a simple verb like Add not added. And for the description try to explain the problem and how it is fixed.

boubaker commented:
The issue has been postponed to post TC, thus I close the issue to be worked in dedicated FT.
